### PR TITLE
Be less intrusive when writing back in the config file

### PR DESCRIPTION
**Pull request by andreia-oca**
# Pull Request Template

## Type of change

-   [ ] 🍕 New feature
-   [ ] 🐛 Bug Fix
-   [ ] 🔥 Breaking change
-   [x] 🧑‍💻 Improvement
-   [ ] 📝 Documentation Update

## Description

Be less intrusive when editing the `genezio.yaml`.

This PR changes the `util` functions used to add new components such as backend, frontend, ssr to:
* Save the comments from the `genezio.yaml`
* Add fields only if they are not set - if the fields are set in the `genezioyaml` and they are conflicting with what we "analyzed", we will prioritize the value from `genezio.yaml`

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [ ] I have updated the documentation;
-   [ ] I have added tests;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Review by costinsin — APPROVED** [review]
LVGTM (looks very good to me)

File: src/commands/deploy/utils.ts [hunk 1]
```diff
@@ -201,17 +198,37 @@ function getNoTargetPackage(errorMessage: string): string | null {
...
+        if (scripts.deploy && typeof scripts.deploy === "string") {
+            scripts.deploy = [scripts.deploy];
+        }
+        if (scripts.build && typeof scripts.build === "string") {
+            scripts.build = [scripts.build];
+        }
+        if (scripts.start && typeof scripts.start === "string") {
+            scripts.start = [scripts.start];
+        }
+    }
```
Comment (cristim67) [inline src/commands/deploy/utils.ts:225]: There’s a duplicated code segment here, matching lines 10-24 in `analyze/utils.ts`. If this logic is likely to stay consistent, consider extracting it into a reusable function to improve maintainability.

**Review by cristim67 — APPROVED** [review]
LGTM

File: src/commands/deploy/utils.ts (hunk 1 above)
Comment (andreia-oca) [reply src/commands/deploy/utils.ts]: Done ✅

**Pull request merged**